### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

File: examples/merkle_update/merkle_update.ml [hunk 1]
```diff
@@ -77,7 +77,7 @@ let input () =
 ;;
 
 (* Now, we write the computation that we want our snark to certify.
-   A [Checked] computation is paramterized by two types. [Checked.t] is
+   A [Checked] computation is parametrized by two types. [Checked.t] is
```
Comment (bkase) [inline examples/merkle_update/merkle_update.ml]: This should be `parameterized`, I think.

File: src/camlsnark_c/libsnark-caml/README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ For announcements and discussions, see the [libsnark mailing list](https://group
 [TOC]
```
Comment (bkase) [inline src/camlsnark_c/libsnark-caml/README.md]: this is part of libsnark, could you open a PR to fix the typos there?

File: src/camlsnark_c/libsnark-caml/doxygen.conf [hunk 3]
```diff
@@ -40,7 +40,7 @@ PROJECT_NUMBER         =
 
 PROJECT_BRIEF          =
```
Comment (bkase) [inline src/camlsnark_c/libsnark-caml/doxygen.conf]: this file as well, could you open a PR on their repo, and revert the changes here?

File: src/camlsnark_c/libsnark-caml/README.md (hunk 2 above)
Comment (rex4539) [reply src/camlsnark_c/libsnark-caml/README.md]: https://github.com/scipr-lab/libsnark/pull/128

File: src/camlsnark_c/libsnark-caml/doxygen.conf (hunk 3 above)
Comment (rex4539) [reply src/camlsnark_c/libsnark-caml/doxygen.conf]: https://github.com/scipr-lab/libsnark/pull/128
